package.json

```diff
@@ -71,6 +71,7 @@
     "gulp-watch": "^4.3.5",
     "jest": "^19.0.2",
     "mock-stdin": "^0.3.0",
+    "sinon": "^1.17.7",
```

Comment:
Re 3, maybe you should not test for Path but rather test that a script from bin can be executed?